ui/main/src/app/modules/feed/feed.component.ts

```diff
@@ -26,26 +27,32 @@ export class FeedComponent implements OnInit {
 
     lightCards$: Observable<LightCard[]>;
     selection$: Observable<string>;
+    totalNumberOfLightsCards : number = 0;
 
 
-    constructor(private store: Store<AppState>) {
+    constructor(private store: Store<AppState>,private lightCardsService: LightCardsService ) {
     }
 
     ngOnInit() {
-        this.lightCards$ = this.store.pipe(
-            select(feedSelectors.selectSortedFilteredLightCards),
-            delay(0), // Solve error: 'Expression has changed after it was checked' --> See https://blog.angular-university.io/angular-debugging/
-            map(lightCards => lightCards.filter(lightCard => !lightCard.parentCardId)),
-            catchError(err => of([]))
-        );
+
         this.selection$ = this.store.select(feedSelectors.selectLightCardSelection);
 
         moment.updateLocale('en', { week: {
             dow: 6, // First day of week is Saturday
             doy: 12 // First week of year must contain 1 January (7 + 6 - 1)
         }});
+
+        
+        this.lightCards$ = this.lightCardsService.getFilteredAndSortedLightCards().pipe(
+            delay(0), // Solve error: 'Expression has changed after it was checked' --> See https://blog.angular-university.io/angular-debugging/
+            map(cards => {
+                this.totalNumberOfLightsCards = cards.length;
+                return cards.slice(0, 100)
```

Comment:
It would be better to make that "100" the same variable as the threshold for display in the template below.

Reply:
ok